coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 92.76
+    "covered_percent": 88.24
```

Comment:
As far as I can tell, this is accurate. It seems like it actually may have been a little while since anyone ran the specs though. So...